Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIB_TIA_Feed/test_data/example.json

```
@@ -680,5 +680,34 @@
         "timeLineData": [],
         "title": "CVE-2021-27152",
         "type": "cve"
+    },
+    "ioc/common":{
+        "id": "6d44a7ed0b826f972e2fb74233c581d80f890552",
```

Or add it to secrets-ignore
```suggestion
        "id": "1111111111111111111111111111111111111111",
```